helm/values/mongodb/values.yaml

```diff
@@ -766,19 +892,21 @@ externalAccess:
     enabled: false
     ## Bitnami Kubectl image
     ## ref: https://hub.docker.com/r/bitnami/kubectl/tags/
-    ## @param externalAccess.autoDiscovery.image.registry Init container auto-discovery image registry
-    ## @param externalAccess.autoDiscovery.image.repository Init container auto-discovery image repository
-    ## @param externalAccess.autoDiscovery.image.tag Init container auto-discovery image tag (immutable tags are recommended)
+    ## @param externalAccess.autoDiscovery.image.registry [default: REGISTRY_NAME] Init container auto-discovery image registry
+    ## @param externalAccess.autoDiscovery.image.repository [default: REPOSITORY_NAME/kubectl] Init container auto-discovery image repository
+    ## @skip externalAccess.autoDiscovery.image.tag Init container auto-discovery image tag (immutable tags are recommended)
+    ## @param externalAccess.autoDiscovery.image.digest Init container auto-discovery image digest in the way sha256:aa.... Please note this parameter, if set, will override the tag
     ## @param externalAccess.autoDiscovery.image.pullPolicy Init container auto-discovery image pull policy
     ## @param externalAccess.autoDiscovery.image.pullSecrets Init container auto-discovery image pull secrets
     ##
     image:
       registry: docker.io
       repository: bitnami/kubectl
-      tag: 1.23.6-debian-10-r8
+      tag: 1.30.2-debian-12-r0
+      digest: ""
```

Comment:
Is there specific use of this digest here?

Reply:
No, It was added in current helm_chart. So added it and kept empty.